pkg/block/fetcher.go

```diff
@@ -379,14 +383,15 @@ func NewTimePartitionMetaFilter(MinTime, MaxTime model.TimeOrDurationValue) *Tim
 }
 
 // Filter filters out blocks that are outside of specified time range.
-func (f *TimePartitionMetaFilter) Filter(metas map[ulid.ULID]*metadata.Meta, synced GaugeLabeled, _ bool) {
+func (f *TimePartitionMetaFilter) Filter(ctx context.Context, metas map[ulid.ULID]*metadata.Meta, synced GaugeLabeled, _ bool) error {
```

Comment:
The convention is to call parameters which are not used `_`. Yes, it still compiles if you have two variables with that name. :-)

Reply:
thanks, resolved now.